backend/models/userTasks.js

```diff
@@ -1,38 +1,38 @@
 const mongoose = require("mongoose");
 
 const taskSchema = mongoose.Schema(
-    {
-        title: {
-            type: String,
-            required: true
-        },
-        user_id: {
-            type: mongoose.Schema.Types.ObjectId,
-            required: true
-        },
-        description: {
-            type: String,
-            required: true
-        },
-        startDate: {
-            type: Date,
-            default: Date.now(),
-            required: true
-        },
-        endDate: {
-            type: Date,
-            required: true
-        },
-        isStarted: {
-            type: Boolean,
-            required: true
-        },
+  {
+    title: {
+      type: String,
+      required: true,
     },
-    {
-        timestamps: true
-    }
+    user_id: {
+      type: mongoose.Schema.Types.ObjectId,
+      required: true,
+    },
+    description: {
+      type: String,
+      required: true,
+    },
+    startDate: {
+      type: Date,
+      default: Date.now(),
+      required: true,
+    },
+    endDate: {
+      type: Date,
+      required: true,
+    },
+    isStarted: {
+      type: Boolean,
+      required: true,
+    },
+  },
+  {
+    timestamps: true,
+  }
 );
-
+// console.log("Heloe");
```

Comment:
Similar to the comment above regarding the use of `console.log` statements, this unnecessary statement should be removed rather than commented.

Reply:
Fixed: Changes are reflected in recent commits